applications/OptimizationApplication/python_scripts/analyses_controller.py

```diff
@@ -10,7 +10,7 @@
 # ==============================================================================
 
 # additional imports
-from KratosMultiphysics.OptimizationApplication.execution_policies.execution_policy_wrapper import ExecutionPolicyWrapper
+from KratosMultiphysics.OptimizationApplication.utilities.execution_policy_decorator import ExecutionPolicyDecorator
```

Comment:
Please change the name of ExecutionPolicyDecorator such that it gets self-explaining. Something like AnalysisWrapper. Policy and Decorator do not sound good.

Reply:
addressed in my general answer